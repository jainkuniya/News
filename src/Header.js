import React, { Component } from 'react';
import { View, Text, StyleSheet } from 'react-native';
import { styles } from './src/common/styles'

const moreStyles = {
	headerTextStyle: {
		fontSize: 20,
		color: 'darkgrey'
	}
}
export default class News extends Component {
	
	render() {
		return(
			<View style={styles.cardView}>
				<Text style={moreStyles.headerTextStyle}>News</Text>
			</View>
		);
	}
}