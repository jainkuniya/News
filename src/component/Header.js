import React, { Component } from 'react';
import { View, Text, StyleSheet } from 'react-native';

const styles = {
	headerViewStyle: {
		backgroundColor: '#F8F8F8',
	    justifyContent: 'center',
	    alignItems: 'center',
	    height: 40,
	    shadowColor: '#000',
	    shadowOffset: { width: 0, height: 2 },
	    shadowOpacity: 0.2,
	    elevation: 2,
	    position: 'relative'
	},
	headerTextStyle: {
		fontSize: 20,
		color: 'darkgrey'
	}
}
export default class News extends Component {
	
	render() {
		const {headerViewStyle, headerTextStyle} = styles;

		return(
			<View style={headerViewStyle}>
				<Text style={headerTextStyle}>News</Text>
			</View>
		);
	}
}