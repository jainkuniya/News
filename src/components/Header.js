import React, { Component } from 'react';
import { View, Text, StyleSheet } from 'react-native';
import styles from '../common/styles';

const moreStyles = StyleSheet.create({
	headerViewStyle: {
		height: 40,
		position: 'relative',
	},
	headerTextStyle: {
		fontSize: 20,
		color: 'darkgrey'
	},
});

export default class News extends Component {
	
	render() {
		const { headerViewStyle, headerTextStyle } = moreStyles;

		return (
			<View style={[styles.cardView, headerViewStyle]}>
				<Text style={headerTextStyle}>News</Text>
			</View>
		);
	}
}
