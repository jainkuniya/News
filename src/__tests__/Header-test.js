import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../components/Header';

test('renders correctly', () => {
  const tree = renderer.create(
    <Header />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
