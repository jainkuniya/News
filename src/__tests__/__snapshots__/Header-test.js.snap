// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#F8F8F8",
        "elevation": 2,
        "justifyContent": "center",
        "shadowColor": "#000",
        "shadowOffset": Object {
          "height": 2,
          "width": 0,
        },
        "shadowOpacity": 0.2,
      },
      Object {
        "height": 40,
        "position": "relative",
      },
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "darkgrey",
        "fontSize": 20,
      }
    }
  >
    News
  </Text>
</View>
`;
